code-challenges.js

```diff
@@ -9,67 +9,248 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
+//setting uyp fonction that takes temp var as argument 
+//returns string of "${temp var} is below boiling point"
+//or returns string of "${temp var} is above boiling point"
+//or returns string of "${temp var} is at boiling point"
+
+
+
+
 var temp1 = 85
 var temp2 = 350
 var temp3 = 212
 
 
+const tempChecker = (temp) =>{
+    if (temp <= 211)
```

Comment:
code will still run just fine without curly braces `{}` but for debugging purposes curly braces would be helpful 
```suggestion
    if (temp <= 211) {
        return `${temp} is below boiling point`
    }...
```